tests/airgapped/airgap.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+
```

Comment:
Can we have a description of what is being done in this script.

Reply:
done